boysen_a07b.js

```diff
@@ -0,0 +1,127 @@
+/*
+Bill Boysen
+boysen_a07b.js
+INFO_2124_WW
+Thoendel
+2/2/2020
+*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "cheese", "milk", "pizza");        
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    //Checks for Q or q to initiate quit sequence
+    if (text == "q" || text == "Q") {

```

Comment:
Great comparisons! You could use a switch statement here instead of an if/else - just another option.